### DISCUSSION TITLE
Grab a field value of an array in POST Request payload

### DISCUSSION THREAD

**Discussion by joychester** [mockoon/mockoon#762] [Help] [answered]
This maybe a question rather than a feature request, since i did not see any places showing how to achieve this:
Given an array of objects in a POST request payload, for example:
```
[
    {
        "id": "101",
        "status": "ACTIVE",
        "name": "TestCamp 2000101"
    },
    {
        "id": "102",
        "status": "ACTIVE",
        "name": "TestCamp 2000102"
    }
]
```
**Question**: 
If I want to grab the value of both id fields, what should i do in mockoon templating? Thank you in advance!!

**Comment by 255kb** [comment, marked as answer]
You cannot get these elements in a single array directly with the `body` or `bodyRaw` helpers. The syntax is based on [object-path](https://www.npmjs.com/package/object-path) and only supports paths like this: `0.id` or `1.id`.

But depending on what you want to achieve you can get each of them: 

```
// get ids one by one
{{#each (bodyRaw)}}
  {{id}},
{{/each}}
```

**Reply by joychester** [reply]
Thank you very much, I was trying to use @index, sounds like I am doing "over-complex" :)
